skimage/io/collection.py

```diff
@@ -73,19 +73,13 @@ def alphanumeric_key(s):
 
 
 class ImageCollection(object):
-
     """Load and manage a collection of image files.
 
-    Note that files are always stored in alphabetical order. Also note that
-    slicing returns a new ImageCollection, *not* a view into the data.
-
     Parameters
     ----------
     load_pattern : str or list
         Pattern glob or filenames to load. The path can be absolute or
-        relative.  Multiple patterns should be separated by os.pathsep,
-        e.g. '/tmp/work/*.png:/tmp/other/*.jpg'.  Also see
-        implementation notes below.
+        relative. Also see implementation notes below.
```

Comment:
I think we should include all possible inputs here, including the pathsep. As far as I know we don't intend to deprecate that? We could keep the original message but include the statement about windows vs *nix, and also the option to pass in a list of patterns.

Reply:
They still work; I tried to state that in the documentation. Is it good?